modules/sdk/gradle-plugins-poshi-runner/src/main/java/com/liferay/gradle/plugins/poshi/runner/PoshiRunnerPlugin.java

```diff
@@ -274,6 +285,25 @@ public void execute(Task task) {
 
 			});
 
+		PluginContainer pluginContainer = project.getPlugins();
+
+		pluginContainer.withId(
+			"com.liferay.test.integration",
+			new Action<Plugin>() {
+
+				@Override
+				public void execute(Plugin plugin) {
+
```

Comment:
SF, don't add an empty line here. We have an empty line between the signature and the body of a method only if the signature is more than 1 line long.